### PR TITLE
Support multiple viselements (#33)

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Solve Issue #33

### CODE REVIEW

File: src/gui/command.ts [hunk 1]
```diff
@@ -120,36 +120,36 @@ export class FindElementsFileCommand {
...
         }
         pythonPath = pythonPath || workspace.getConfiguration("python").get("defaultInterpreterPath", "python");
         window.withProgress(
             {
                 location: ProgressLocation.Notification,
                 cancellable: false,
-                title: l10n.t("Finding visual element descriptors file"),
+                title: l10n.t("Finding visual element descriptor files"),
             },
             async (progress) => {
```
Comment (FabienLelaquais) [inline src/gui/command.ts]: Is this 'progress' used anywhere?

File: src/assets/find_element_file.py [hunk 2]
```diff
@@ -4,10 +4,17 @@
...
 if util.find_spec("taipy") and util.find_spec("taipy.gui"):
     from taipy.gui import Gui
-    element_file_path = f"{os.path.dirname(inspect.getfile(Gui))}{os.sep}webapp{os.sep}viselements.json"
-    if os.path.exists(element_file_path):
-        print(f"Path: {element_file_path}")
+
+    taipy_path = f"{os.path.dirname(os.path.dirname(inspect.getfile(Gui)))}"
+    potential_file_paths = [
+        f"{taipy_path}{os.sep}gui{os.sep}viselements.json",
+        f"{taipy_path}{os.sep}gui_core{os.sep}viselements.json",
```
Comment (FredLL-Avaiga) [inline src/assets/find_element_file.py:11]: good first step

**Review by FredLL-Avaiga — APPROVED** [review]
😍

File: src/gui/command.ts [hunk 3]
```diff
@@ -120,36 +120,36 @@ export class FindElementsFileCommand {
                 workspace.workspaceFolders?.map(({ uri }) => uri.fsPath),
             );
         } catch (error: any) {
-            getLog().info(l10n.t("Find element descriptors file: "), error.message || error);
+            getLog().info(l10n.t("Find element descriptor files: "), error.message || error);
         }
         pythonPath = pythonPath || workspace.getConfiguration("python").get("defaultInterpreterPath", "python");
         window.withProgress(
```
Comment (FabienLelaquais) [inline src/gui/command.ts:126]: I don't understand the logic of this piece of code...
Whether we find no element descriptor files, or fail to find one, or...

**Review by FabienLelaquais — APPROVED** [review]
Looks good!

File: src/gui/command.ts (hunk 1 above)
Comment (dinhlongviolin1) [reply src/gui/command.ts]: it is not, just the default parameter of the returned function, we can make it private

File: src/gui/command.ts (hunk 3 above)
Comment (dinhlongviolin1) [reply src/gui/command.ts:126]: do you mean the with progress section? basically there is a window popping up stating it is finding the visual element descriptor. I can show you in the daily

**Pull request merged**